.github/ISSUE_TEMPLATE/bugs-performance-issues.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report (including performance and build issues)
+about: Help us improve the GPflow code base
+labels: bug
+---
+
+<!-- Lines like this are comments and will be invisible -->
+
+# Bug / performance issue / build issue
+
+<!-- A clear and concise description of what the bug is. -->
+
+## To reproduce
+
+**Minimal, reproducible example**
+<!-- We need to be able to reproduce the bug by simply copy&pasting your code -->
+```python
+# This is the place for your code that reproduces the bug.
+# Please make sure it does not depend on external libraries, specific datasets, etc., and the smaller, the better :)
```

Comment:
what about external libraries that GPflow uses? or do you not consider those external?

Reply:
The intent was to get people's MREs not to require *additional* dependencies beyond gpflow (and gpflow's own dependencies)

Reply:
in that case, how about, instead of

> Please make sure it does not depend on external libraries

we have

>  Please make sure it does not depend on libraries that aren't also dependencies of GPflow

Reply:
```suggestion
# Please make sure it does not depend on external libraries (beyond GPflow's own requirements), specific datasets, etc., and the smaller, the better :)
```

Reply:
sounds good